### PR TITLE
Add retry for storageSystem check

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes: #10637

### CODE REVIEW

File: ocs_ci/ocs/resources/storage_cluster.py [hunk 1]
```diff
@@ -937,6 +937,7 @@ def verify_ocs_csv(ocs_registry_image=None):
             )
 
 
+@retry(AssertionError, 15, 10, 0)
```
Comment (vavuthu) [inline ocs_ci/ocs/resources/storage_cluster.py:940]:
```suggestion
@retry(AssertionError, 15, 10, 1)
```
if backoff is 0, then delay will be 0 from 2nd iteration

**Review by ebenahar — APPROVED** [review]

File: ocs_ci/ocs/resources/storage_cluster.py (hunk 1 above)
Comment (petr-balogh) [reply ocs_ci/ocs/resources/storage_cluster.py]: Good catch, fixed

**Review by dahorak — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Comment by petr-balogh** [comment]
Verified here:
https://url.corp.redhat.com/800116c

**Pull request merged**